tests/subsys/logging/log_syst/prj.conf

```diff
@@ -0,0 +1,21 @@
+CONFIG_LOG=y
+CONFIG_ASSERT=n
+CONFIG_ZTEST=y
+CONFIG_LOG_BACKEND_MOCK=y
+CONFIG_LOG_BACKEND_UART=n
+CONFIG_SOC_LOG_LEVEL_OFF=y
+CONFIG_ARCH_LOG_LEVEL_OFF=y
+CONFIG_KERNEL_LOG_LEVEL_OFF=y
+CONFIG_LOG_RUNTIME_FILTERING=n
+CONFIG_MAIN_THREAD_PRIORITY=5
```

Comment:
Do you really need to set the main thread priority ?

Reply:
Removed the CONFIG_MAIN_THREAD_PRIORITY for now.